docker/Dockerfile

```
@@ -70,11 +69,14 @@ RUN \
     ldconfig && \
     # Install mongodb driver from source
     cd /opt && \
-    curl -kOL https://github.com/mongodb/mongo-cxx-driver/archive/legacy-1.1.2.tar.gz && \
-    tar xfz legacy-1.1.2.tar.gz && \
-    cd mongo-cxx-driver-legacy-1.1.2 && \
-    scons --use-sasl-client --ssl && \
-    scons install --prefix=/usr/local --use-sasl-client --ssl && \
+    wget https://github.com/mongodb/mongo-c-driver/releases/download/1.16.0/mongo-c-driver-1.16.0.tar.gz && \
+    tar xfvz mongo-c-driver-1.16.0.tar.gz && \
+    cd mongo-c-driver-1.16.0 && \
+    mkdir cmake-build && \
+    cd cmake-build && \
+    cmake3 -DENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF ..   && \
```

```suggestion
    cmake -DENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF ..   && \
```

According my tests, cmake works in CentOS 8 (eg. https://github.com/telefonicaid/fiware-orion/blob/feature/3132_poc_isolating_driver_usage/ci/rpm8/build-dep.sh#L76)